.github/workflows/docker.yaml

```
@@ -64,7 +64,7 @@ jobs:
       - name: push latest
         if: ${{ github.event_name == 'push' && github.ref_name == 'main' }}
         run: |
-          docker tag $IMAGE_NAME:${{ github.sha }} $IMAGE_NAME:$VERSION
-          docker tag $IMAGE_NAME:${{ github.sha }} $IMAGE_NAME:latest
-          docker push $IMAGE_NAME:$VERSION
-          docker push $IMAGE_NAME:latest
\ No newline at end of file
+          docker tag $IMAGE_NAME:${{ github.sha }} $DOCKER_MAIN/$IMAGE_NAME:$VERSION
+          docker tag $IMAGE_NAME:${{ github.sha }} $DOCKER_MAIN/$IMAGE_NAME:latest
+          docker push $DOCKER_MAIN/$IMAGE_NAME:$VERSION
+          docker push $DOCKER_MAIN/$IMAGE_NAME:latest
```

New line at end of file
```suggestion
          docker push $DOCKER_MAIN/$IMAGE_NAME:latest

```